docs/plugins/automelt.rst

```diff
@@ -6,13 +6,41 @@ automelt
     :tags: untested fort productivity items stockpiles
     :no-command:
 
-When `enabled <enable>`, this plugin adds an option to the :kbd:`q` menu for
-stockpiles. When the ``automelt`` option is selected for the stockpile, any
-items placed in the stockpile will automatically be designated to be melted.
+Automelt checks monitor-enabled stockpiles once every in-game day, and will mark valid items for melting.
+
+Please see `gui/automelt` for the interactive configuration dialog.
 
 Usage
 -----
 
 ::
 
     enable automelt
+    automelt [status]
+    automelt (designate)
```

Comment:
no need for parens around single literal strings. just `automelt designate` is fine

Reply:
Fixed in latest